decomposer/session_manager.py

```diff
@@ -45,8 +43,10 @@ def get_session(agent_name: str) -> AgentSession:
             sess = SessionManager._sessions.get(agent_name)
             if sess is None:
                 sess = SessionManager._factory.create_session(
-                    "direct", agent_name, "localhost", SessionManager.AGENTS_PORT, False,
-                    {"user_id": os.environ.get("USER")}
+                    "direct",
+                    agent_name,
+                    use_direct=True,
+                    metadata={"user_id": os.environ.get("USER")}
```

Comment:
Turn use_direct on.  When on, this means that any /external_agent references will be called as direct sessions like a library, not over http like a web client.  This was useful when going to the one agent before consolidating the other agents into it.